### PR TITLE
FAQ page rough draft

### PR DESCRIPTION
**Pull request by daveschoening**
First edits on faq page

### CODE REVIEW

**Comment by leavenember** [comment]
What's the plan with the timeline section here?

File: faq.html [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+                        Absolutely! Companies love when applicants already have practical hands-on experience as well as self-made projects!
+                      </div>
+                    </div>
+                  </div>
+              </div>
+
+        </section>
+
+        
+        <!-- Section: Timeline -->
```
Comment (bcbielecki) [inline faq.html:180]: Hi David. It looks like you copied over the timeline from the syllabus page. The elements are hidden from the viewport, but please remove the code anyway. It's extraneous.

**Comment by bcbielecki** [comment]
Hi David! Good work; we're almost done here! Please add the FAQ link to the navbar in the header.js file, and resolve the comments.

**Review by bcbielecki — CHANGES_REQUESTED** [review]
Hi David! Good work; we're almost done here! Please add the FAQ link to the navbar in the header.js file, and resolve the comments.

**Comment by bcbielecki** [comment]
Deprecated. Another PR replace this one.

**Pull request closed without merging**